bigframes/session/__init__.py

```diff
@@ -223,6 +223,17 @@ def _create_and_bind_bq_session(self):
         query_job.result()  # blocks until finished
         self._session_id = query_job.session_info.session_id
 
+        # The anonymous dataset is used by BigQuery to write query results and
+        # session tables. BigQuery DataFrames also writes temp tables directly
+        # to the dataset, no BigQuery Session required. Note: there is a
```

Comment:
Could you please help elaborate "no BigQuery Session required", isn't a session always there in BigQuery DataFrames?

Reply:
It is, but I'm working on removing that. This is a step in that direction. :-)